cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -30,7 +30,9 @@
                 CruiseControl.DISK_CAPACITY_GOAL,
                 CruiseControl.NETWORK_INBOUND_CAPACITY_GOAL,
                 CruiseControl.NETWORK_OUTBOUND_CAPACITY_GOAL,
-                CruiseControl.CPU_CAPACITY_GOAL,
+                // TODO: The CPU metric are currently not reported correctly when running on Kubernetes
+                //       we should add this back in once fixed upstream, see https://github.com/linkedin/cruise-control/issues/1242
```

Comment:
Is there a Strimzi issue connected to this also?

Reply:
That is an interesting point, this is not an issue with Strimzi but with CC its not an "issue" more of a TODO? I can make one if you think it needs tracking?

Reply:
Added #3215